mediainfo@aclon/files/mediainfo@aclon/wget_lang.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# downloads the language file for mediainfo
+SELF_PATH=$(dirname $0)
+pushd $SELF_PATH/po
+lang=$(echo $LANG | sed 's/.UTF-8//')
+lang_=$(echo $lang | sed 's/_/-/')
+wget -O ${lang}.csv https://raw.githubusercontent.com/MediaArea/MediaInfo/master/Source/Resource/Plugin/Language/${lang_}.csv && rm 'zh-CN.csv' || mv -f ${lang_}.csv ${lang}.csv
```

Comment:
Download MediaInfo's official i18n csv, if success then rename `zh_CN.csv` to `zh-CN.csv`.
However I haven't test it on other language like `ca.csv` `ja.csv` etc...